# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/boost/hotfix-siglost.patch.sparc
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- boost_1_86_0/libs/process/src/error.cpp.vanilla	2024-08-27 15:34:47.922313492 +0200
+++ boost_1_86_0/libs/process/src/error.cpp	2024-08-27 15:34:54.926314051 +0200
@@ -112,7 +112,7 @@
 #            if defined(SIGKILL)
                     case SIGKILL: return "SIGKILL:   Kill signal";
 #            endif
-#            if defined(SIGLOST)
+#            if 0 && defined(SIGLOST)
                     case SIGLOST: return "SIGLOST:   File lock lost (unused)";
 #            endif
 #            if defined(SIGPIPE)
